docs/documentation/maya/ui_overview.md

```diff
@@ -99,20 +99,20 @@ Option boxes might be used for deformer creation. Openning an auxiliar window th
 - **Remove Deform Mesh**. Removes a [Deform Mesh](simshape.md#requirements) from the deformer selected. The selection must consist of: first the Deform Mesh that we want to remove and lastly the mesh with the deformer applied.
```

Comment:
I know this information might be redundant as it is exposed in the table but we might want to also add the information related to adding the collider and removing the collider